RELEASE-NOTES.md

```diff
@@ -1,25 +1,126 @@
 # Riak CS 2.1.0 Release Notes
 
+## General Information
+
+This is a new release including new garbage collection, metrics system
+and several other new features. Riak CS 2.1 is supposed to work with
+both Riak 2.0.5 and 2.1.1.
+
 ## Additions
 
-- Module-level hook point for limiting user access and quota usage has
-  been introduced, with very preliminary simple node-wide limting
-  example modules. Operators can make, plug in or combine different
-  modules as quota-limiting, rate-limiting or bandwidth-limiting
-  depeding on their unique requirements. This is **experimental** and
-  plugin SPI (service provider interface) may or may not change in
-  future effort to improve this
-  system. [riak_cs/#1118](https://github.com/basho/riak_cs/pull/1118)
+- Add several options to manual garbage collection batch command, for
+  more flexible operation and to address race condition between
+  replication and garbage collection. Specifically, `--leeway` for
+  temporary leeway period, `--start` and `--end` for target period,
+  `--max-workers` for temporary
+  concurrency. [riak_cs/#1147](https://github.com/basho/riak_cs/pull/1147),
+  [riak_cs/#1148](https://github.com/basho/riak_cs/pull/1148),
+  [riak_cs/#1149](https://github.com/basho/riak_cs/pull/1149).
+
+- Refactor and rename `riak_cs_gc_d`, which is garbage collection
+  daemon by splitting user-facing GC execution state machine and
+  working GC daemon
+  [riak_cs/#1144](https://github.com/basho/riak_cs/pull/1144).
+
+- Add orphan blocks scanner and collector. In case inconsistencies
+  between manifests and blocks, especially manifests deleted and
```

Comment:
I was wrong. `block_audit` script compares blocks and manifests in manifest buckets. Sorry.